ui/widgets/sidebar/src/components/sidebar-component.tsx

```diff
@@ -149,15 +168,27 @@ const SidebarComponent: React.FC<RootComponentProps> = props => {
   const subtitleUi = useMemo(
     () =>
       loginQuery.data?.id ? (
-        <DidField did={loginQuery.data?.id} textColor="grey7" />
+        <DidField
+          did={loginQuery.data?.id}
+          textColor="grey7"
+          copyLabel={t('Copy to clipboard')}
+          copiedLabel={t('Copied')}
+        />
       ) : (
         <Text variant="footnotes1" customStyle="text-grey5 whitespace-normal" truncate breakWord>
           {t('Connect to see exclusive member only features.')}
         </Text>
       ),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     [loginQuery.data?.id],
   );
 
+  const profileName = useMemo(
+    () => (myProfileQuery.data?.did?.id ? myProfileQuery.data?.name : t('Guest')),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [myProfileQuery.data?.did?.id],
```

Comment:
I think you should add also the `myProfileQuery.data?.name` in the dependency list, for example if I update my name it will still return the old name.

Reply:
also, please don't disable the eslint rules, try to think about the implications first